validator/utils/column_picker.py

```diff
@@ -0,0 +1,68 @@
+import json
+import re
+from logging import getLogger
+
+from fastapi import HTTPException
+
+from validator.core.config import Config
+from validator.core.constants import COLUMN_PICKER_NUM_PREVIEW_ROWS
+from validator.core.constants import SYNTH_MODEL
+from validator.core.constants import SYNTH_MODEL_TEMPERATURE
+from validator.core.models import ColumnPickerResponse
+from validator.core.models import SuitableDataset
+from validator.utils.call_endpoint import post_to_nineteen_ai
+from validator.utils.prompts import load_prompts
+
+
+logger = getLogger(__name__)
```

Comment:
fiber logger porfa

Reply:
This is merging into #138 where the default logger gets the formatting and log info from the fiber logger. So effectively it is the same as the fiber logger but we add context tags to the file handler